src/BuildScriptGeneratorCli/Commands/Binders/BuildScriptCommandBinder.cs

```diff
@@ -33,25 +33,25 @@ public class BuildScriptCommandBinder : BuildCommandBaseBinder<BuildScriptComman
             Option<bool> debugMode)
             : base(sourceDir, platform, platformVersion, package, osRequirements, appType, buildCommandFile, compressDestinationDir, property, dynamicInstallRootDir, logPath, debugMode)
         {
-            this.buildScriptOutputOption = buildScriptOutputOption;
+            this.buildScriptOutput = buildScriptOutput;
         }
 
         protected override BuildScriptCommandProperty GetBoundValue(BindingContext bindingContext) =>
             new BuildScriptCommandProperty
             {
-                OutputPath = bindingContext.ParseResult.GetValueForOption(this.buildScriptOutputOption),
-                SourceDir = bindingContext.ParseResult.GetValueForArgument(this.sourceDir),
-                PlatformName = bindingContext.ParseResult.GetValueForOption(this.platform),
-                PlatformVersion = bindingContext.ParseResult.GetValueForOption(this.platformVersion),
-                ShouldPackage = bindingContext.ParseResult.GetValueForOption(this.package),
-                OsRequirements = bindingContext.ParseResult.GetValueForOption(this.osRequirements),
-                AppType = bindingContext.ParseResult.GetValueForOption(this.appType),
-                BuildCommandsFileName = bindingContext.ParseResult.GetValueForOption(this.buildCommandFile),
-                CompressDestinationDir = bindingContext.ParseResult.GetValueForOption(this.compressDestinationDir),
-                Properties = bindingContext.ParseResult.GetValueForOption(this.property),
-                DynamicInstallRootDir = bindingContext.ParseResult.GetValueForOption(this.dynamicInstallRootDir),
-                LogFilePath = bindingContext.ParseResult.GetValueForOption(this.logPath),
-                DebugMode = bindingContext.ParseResult.GetValueForOption(this.debugMode),
+                OutputPath = bindingContext.ParseResult.GetValueForOption(this.buildScriptOutput),
+                SourceDir = bindingContext.ParseResult.GetValueForArgument(this.SourceDir),
+                PlatformName = bindingContext.ParseResult.GetValueForOption(this.Platform),
+                PlatformVersion = bindingContext.ParseResult.GetValueForOption(this.PlatformVersion),
+                ShouldPackage = bindingContext.ParseResult.GetValueForOption(this.Package),
```

Comment:
Could you please clarify if you see any issues with reconciling the names together?
OutputPath vs. buildScriptOutput
PlatformName vs Platform
etc.

Reply:
Sorry i forgot to change the outputPath name. This one is aligned now.
For PlatformName and Platform, the property name used in `BuildScriptCommandBinder` is PlatformName, but in `BuildCommandBaseBinder` where it was inherited from, it uses Platform as the variable name. In that case, i kept the different names here.

Reply:
Thanks for your reply. Is it normal that `this.SourceDir` and others are capitalized now? I thought that here we would be referring to the class member that are lowercase. Am I missing something?
Similarly, could you please clarify if for the other names we still should keep them as non-reconciled? I'm thinking about ShouldPackage vs. Package, BuildCommandsFileName vs. BuildCommandFile, Properties vs. Property, LogFilePath vs. LogPath.
Even if they are called that way in another class, would it be expensive to update them?

Reply:
Yes, so for instance, `SourceDir` at the start of line 43 is refering to the properties from the property file. Here, I kept the naming convention the same as we have in command files. Later in the line, `this.SourceDir` refers to the arguments defined from its inherited file `BuildCommandBaseBinder`. Because we changed ways of defining options and arguments in base classes from protected fields to protected properties, we need to capitalize them. But for `outputPath`, because it is a private field defined in this level, it needs to be in small letters.

And for changing some of the others, because lots of them are shared by multiple inherited docs, some of the names might change from one to another in different files and also a bit time consuming to change. I would prefer to keep them as it is for now.

Reply:
For the sake of consistency, I would prefer to update all of the Binders to use properties so that we don't run into any confusion in the future for why the different options are capitalized differently. You'll notice that even now, we are seeing a warning about converting to readonly, which can be an indication that something is a bit off.

![image](https://user-images.githubusercontent.com/107068277/223767882-dfd5c21d-32f5-4419-b365-912fec9a8808.png)

You can use `ctrl+R+R` when renaming these, which should find all references to each field across all files (_though, I'll note that all of these are private methods, so it would be fishy if there were references to them outside of a single file_), and update to whatever the new name is:
![image](https://user-images.githubusercontent.com/107068277/223768669-6697f5ea-df7d-4015-8230-e608df80998d.png)
Then you can just add a `{ get; set; }` and call it a day